src/main/java/uk/gov/hmcts/ccd/domain/model/std/validator/ValidationError.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.ccd.domain.model.std.validator;
+
+public class ValidationError {
+
+    public static final String ARGUMENT_INVALID = "Input not valid";
+    public static final String PARTIES_INVALID = "At least two fields must be defined for a party";
+    public static final String SORT_BY_INVALID = "Sort by category invalid";
+    public static final String SORT_DIRECTION_INVALID = "Sort direction invalid";
+    public static final String MAX_RECORD_COUNT_INVALID = "Max return record count invalid";
```

Comment:
It would be good to provide slightly more descriptive error messages when possible - example for this one:
```suggestion
    public static final String MAX_RECORD_COUNT_INVALID = "Max return record count must be between 0 and 10000";
```